packages/oidc-middleware/test/e2e/harness/views/login.ejs

```diff
@@ -5,10 +5,10 @@
         font-family: Helvetica Neue,Helvetica,Arial,sans-serif;
       }
     </style>
-    <script src="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/1.13.0/js/okta-sign-in.min.js" type="text/javascript"></script>
-    <link href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/1.13.0/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+    <script src="<%= cdnUrl %>/js/okta-sign-in.min.js" type="text/javascript"></script>
+    <link href="<%= cdnUrl %>/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
     <!-- Optional, customizable css theme options. Link your own customized copy of this file or override styles in-line -->
-    <link href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/1.13.0/css/okta-theme.css" type="text/css" rel="stylesheet"/>
```

Comment:
`okta-theme.css` have been removed in `3.x`. need a /smart way/ to determine whether loads the file.

Reply:
I was aware of that but this didn’t throw any errors. So, took the short cut. Will check if there’s a way to comment that line out for 3.x versions.

Reply:
Actually, since we will use only 3.x versions both for global and cell CDNs, I can safely remove this. What do you think, @haishengwu-okta ?

Reply:
sounds good to me. Maybe just comment it out by adding comments about why we do it?